subsys/zigbee/cli/zigbee_cli_cmd.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <version.h>
+#include <string.h>
+#include <shell/shell.h>
+#include <power/reboot.h>
+
+#include <zboss_api.h>
+#include <zb_error_handler.h>
+#include <zb_version.h>
+#include "zigbee_cli.h"
+#include "zigbee_cli_utils.h"
+
+#define DEBUG_HELP \
+	"Return state of debug mode.\n"
+
+#define DEBUG_ON_HELP \
+	"Turn on debug mode.\n"
+
+#define DEBUG_OFF_HELP \
+	"Turn off debug mode.\n"
+
+#define DEBUG_WARN_MSG \
+	"You are about to turn the debug mode on. This unblocks several\n" \
+	"additional commands in the CLI. They can render the device unstable.\n" \
+	"It is implied that you know what you are doing."
+
+/**@brief Print CLI, ZBOSS and Zephyr kernel version
+ *
+ * @code
+ * version
+ * @endcode
+ *
+ * @code
+ * > version
+ * CLI: Jul 2 2020 16:14:18
+ * ZBOSS: 3.1.0.59
+ * Done
+ * @endcode
+ */
+static int cmd_version(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	shell_print(shell, "CLI: " __DATE__ " " __TIME__);
+	shell_print(shell, "ZBOSS: %d.%d.0.%d", ZBOSS_MAJOR, ZBOSS_MINOR,
+		    ZBOSS_SDK_REVISION);
+	shell_print(shell, "Zephyr kernel version: %s", KERNEL_VERSION_STRING);
+
+	zb_cli_print_done(shell, false);
+	return 0;
+}
+
+/**@brief Perform device reset using `sys_reboot()`.
+ *
+ * @code
+ * > reset
+ * @endcode
+ */
+static int cmd_reset(const struct shell *shell, size_t argc, char **argv)
```

Comment:
What is purpose of this command in this module?
It is already implemented here: subsys/bluetooth/mesh/shell.c
Unfortunately, we do not have any chance to detect commands collision.

Reply:
Yes, that's a good idea